end2end/tests/reportBuilder.spec.ts

```diff
@@ -2,30 +2,238 @@ import { test, expect } from '@playwright/test';
+    await createButton.click();
+    await expect(createButton).toBeFocused();
+});
+
+test('Modify search with AND logical filter', async ({ page }) => {
+    await page.goto("https://host.docker.internal/Test_Request_Portal/?a=reports&v=3&query=N4IgLgpgTgtgziAXAbVASwCZJAYwIaQDmA9lAJ4CSAIiADQjEAO0Bp2AvHSDATgBbYAZqRgB9AKwQ8ABgDsXQgQjYAggDkaAX1rosiEBggAbCJCz0mLMG32d6PMPyTT6iyKo0hNAXXoArYjQAOwQUXxA4UjAkYG0QQlMqAjwkZBAAFi4AZhBwozQYNGjEAEZpcvp8wrAAeUFBOFNnTSA&indicators=NobwRAlgdgJhDGBDALgewE4EkAiYBcYyEyANgKZgA0YUiAthQVWAM4bL4AMAvpeNHCRosuAonhFUUAEIBXZGijNaDfGGZt0HPDz6RYCFBhxqALMvqMwAWVkkiAAhLQyD5GQAeHapu27%2BBkLGomAAzBaqBADK0ADm5E4ubp7erOxc3AC6QA%3D%3D");
+    await page.getByRole('button', { name: 'Modify Search' }).click();
+    await page.getByLabel('add logical and filter').click();
+    await page.getByRole('cell', { name: 'Resolved' }).locator('a').click();
+    await page.getByRole('option', { name: 'Submitted', exact: true }).click();
+    await page.getByRole('button', { name: 'Next Step' }).click();
+    await page.locator('#indicatorList').getByText('General Form').click();
+    await page.getByTitle('indicatorID: 7\nRadio').locator('span').click();
+    await page.getByRole('button', { name: 'Generate Report' }).click();
+    const updatedFilter = page.getByLabel('Sort by Radio');
+    await expect(updatedFilter).toBeVisible();
+    await page.getByRole('button', { name: "Edit Labels" }).click();
+    await page.getByRole('button', { name: 'Save Change' }).click();
+});
 
-  await expect(page.locator('th').nth(4)).toContainText('Numeric');
+test("Verify multi line textbox", async ({ page }) => {
+    await page.goto("https://host.docker.internal/Test_Request_Portal/?a=reports&v=3&query=N4IgLgpgTgtgziAXAbVASwCZJHSAHASQBEQAaEAez2gEMwKpsBeMkGOgYwAtsoI4KAGwBuELOQDmdCNgCCAORIBfUuiyIQHaRIYBPYqyq16jDS3Lsw3bADMGMAPoBWCDQAMAdlZTIcxSBU1bAwIQQhIcUpqKDoGZlZLa0Q3SWk%2FZQBdcgArCjQAOwQUEAK0MDRYqHkaGBksnAYwJGAVEAlwojoaJGQQABZWL3IAZhB6wTQYMqQARjd58gmpsAB5Gxs4cKQ3JSA%3D%3D&indicators=NobwRAlgdgJhDGBDALgewE4EkAiYBcYyEyANgKZgA0YUiAthQVWAM4bL4AMAvpeNHCRosuAtGIQUGZrQb4wzNug54efSLARSR8xPCKooAIQCuyNFBn1GC6kpVr%2BmoRhzyALFbkEAsiZJEAAQk0GSByGQAHhx27Fy8ToLabgQA7F42AEqIcKiKcaoJGknCKWAAzBnyAMrQAObkwaHhUTGsBTwAukA");
+    const firstText = page.getByRole("cell", { name: "12977", exact: true });
+    await firstText.click();
+    const multiText = page.getByLabel("Multi line text", { exact: true });
+    await multiText.click();
+    await multiText.fill("This is the text");
+    await page.getByRole("button", { name: "Save Change" }).click();
+    const changedText = page.getByRole('cell', { name: "This is the text" });
+    await expect(changedText).toBeVisible();
+    await changedText.click();
+    await multiText.click();
+    await multiText.fill('12977');
+    await page.getByRole("button", { name: 'Save Change' }).click();
+    await expect(firstText).toBeVisible();
 });
+
+test("Select multiple filter using AND/OR,", async ({ page }) => {
```

Comment:
```suggestion
test('Select multiple filter using AND/OR (UI only, does not validate filter results)', async ({ page }) => {
```
As before: if these tests are not going to check the accuracy of resulting data, we don't want to give a false impression. Therefore, clarification is needed to explain this only tests UI components.

Reply:
fixed